domain/usecases/src/main/java/com/chocolate/usecases/direct_message/SendMessageUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package com.chocolate.usecases.direct_message
+
+import com.chocolate.entities.directMessage.DMMessage
+import repositories.DirectMessageRepository
+import java.util.Date
+import javax.inject.Inject
+
+class SendMessageUseCase @Inject constructor(
+    private val directMessageRepository: DirectMessageRepository
+) {
+
+    suspend operator fun invoke(
+        message: String,
+        senderId: String,
+        sentAt: Date,
+        currentChatId: String,
+        currentOrgName: String,
+        senderFullName: String,
+        senderAvatarUrl: String,
+    ) {
```

Comment:
create a model to contain those parameters. the method parameter should not exceed 3 parameters.